src/systems/ec/specs/getting-started/getting-started.md

```diff
@@ -1,3 +1,11 @@
 ## Getting started
 
-Steps
+The Europa Component Library (ECL) is a library of components used in all European Commission websites hosted under ec.europa.eu domain. 
+The library contains all available components that you can use to build your website.
+
+All library elements are accompanied by
+
+*	documentation explaining what the component is intended for and how it should be used
+*	a visual demonstration of the component
+*	HTML/CSS code for implementation
```

Comment:
We will probably update this line on our side

Reply:
Changed it. Sorry for all the troubles.